src/EventListener/UserCartRecalculationListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\ShopApiPlugin\EventListener;
+
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Order\Context\CartContextInterface;
+use Sylius\Component\Order\Context\CartNotFoundException;
+use Sylius\Component\Order\Processor\OrderProcessorInterface;
+use Symfony\Contracts\EventDispatcher\Event;
```

Comment:
```suggestion
use Symfony\Component\EventDispatcher\Event;
```

Or we should probably add contracts to composer

Reply:
Oh, now I see it... https://github.com/Sylius/ShopApiPlugin/pull/682#discussion_r517482564 What would you say for using JWT class explicitly? 

Also, can we maybe make it explicit, that we are removing service from the container (for example in CompilerPass with link to issue on Github) and declare separate service? Or at least link in the comment that is right now, to current issues on Github?